test(x/gamm): add liquidity event tests (#2141)
osmosis-labs/osmosis: x/gamm/spec/README.md

TBD

### `types.TypeEvtPoolJoined`

> This event is emitted after one of `JoinPool` or `JoinPoolNoSwap` joins

Author: ```suggestion
This event is emitted after one of `JoinPool` or `JoinPoolNoSwap` complete joining
```

the requested pool successfully.

It consists of the following attributes:

* `sdk.AttributeKeyModule` - "module"
  * The value is the module's name - "gamm".

> * sdk.AttributeKeySender

Author: ```suggestion
* `sdk.AttributeKeySender`
```

  * The value is the address of the sender who created the swap message.

> * types.AttributeKeyPoolId

Author: ```suggestion
* `types.AttributeKeyPoolId`
```

  * The value is the pool id of the pool where swap occurs.

> * types.AttributeKeyTokensIn

Author: ```suggestion
* `types.AttributeKeyTokensIn`
```
